spec/support/bulk_upload/log_to_csv.rb

```diff
@@ -0,0 +1,254 @@
+      nil,
+      nil,
+      nil, # 60
+      log.prevten,
+      log.prevloc,
+      ((log.ppostcode_full || "").split(" ") || [""]).first,
+      ((log.ppostcode_full || "").split(" ") || [""]).last,
+      previous_postcode_known,
+      log.layear,
+      log.waityear,
+      homeless,
+      log.reasonpref,
+      log.rp_homeless, # 70
+      log.rp_insan_unsat,
+      log.rp_medwel,
+      log.rp_hardship,
+      log.rp_dontknow,
+      cbl,
+      chr,
+      cap,
+      log.referral,
+      log.period,
+
+      log.brent, # 80
+      log.scharge,
+      log.pscharge,
+      log.supcharg,
+      log.tcharge,
+      log.chcharge,
+      log.household_charge,
+      log.hbrentshortfall,
+      log.tshortfall,
+      log.voiddate&.day,
+
+      log.voiddate&.month, # 90
+      log.voiddate&.strftime("%y"),
+      log.mrcdate&.day,
+      log.mrcdate&.month,
+      log.mrcdate&.strftime("%y"),
+      nil,
+      log.startdate&.day,
+      log.startdate&.month,
+      log.startdate&.strftime("%y"),
+      log.offered,
+
+      log.propcode, # 100
+      log.beds,
+      log.unittype_gn,
+      log.builtype,
+      log.wchair,
+      log.unitletas,
+      log.rsnvac,
+      log.la,
+      ((log.postcode_full || "").split(" ") || [""]).first,
+      ((log.postcode_full || "").split(" ") || [""]).last,
+
+      nil, # 110
+      log.owning_organisation&.old_visible_id,
+      nil,
+      log.managing_organisation&.old_visible_id,
+      leftreg,
+      nil,
+      log.incfreq,
+      log.sheltered,
+      log.illness,
+      log.illness_type_1,
+
+      log.illness_type_2, # 120
+      log.illness_type_3,
+      log.illness_type_4,
+      log.illness_type_5,
+      log.illness_type_6,
+      log.illness_type_7,
+      log.illness_type_8,
+      log.illness_type_9,
+      log.illness_type_10,
+      london_affordable_rent,
+
+      intermediate_rent_type, # 130
+      log.irproduct_other,
+      log.declaration,
+      log.joint,
+      renewal,
+      "\n",
+    ].join(",")
+  end
+
+  def renewal
+    case log.renewal
+    when 1
+      1
+    when 0
+      2
+    end
+  end
+
+  def london_affordable_rent
+    case log.renttype
+    when Imports::LettingsLogsImportService::RENT_TYPE[:london_affordable_rent]
+      1
+    end
+  end
+
+  def intermediate_rent_type
+    case log.renttype
+    when Imports::LettingsLogsImportService::RENT_TYPE[:rent_to_buy]
+      1
+    when Imports::LettingsLogsImportService::RENT_TYPE[:london_living_rent]
+      2
+    when Imports::LettingsLogsImportService::RENT_TYPE[:other_intermediate_rent_product]
+      3
+    end
+  end
+
+  def leftreg
+    case log.leftreg
+    when 3
+      3
+    when 1
+      4
+    when 2
+      5
+    when 0
+      6
+    end
+  end
+
+  def net_income_known
+    case log.net_income_known
+    when 0
+      1
+    when 1
+      2
+    when 2
+      4
+    end
+  end
+
+  def previous_postcode_known
+    case log.ppcodenk
+    when 1
+      1
+    when 0
+      2
+    end
```

Comment:
can this be a method? Like `checkbox_value(field)` etc?

Reply:
I find they're always different which is not ideal. look at `cbl` for example which does the inverse...

Reply:
oh i misread, sure

Reply:
done